src/webui/btjson.cpp

```diff
@@ -800,6 +801,7 @@ QVariantMap toMap(BitTorrent::TorrentHandle *const torrent)
     ret[KEY_TORRENT_NUM_INCOMPLETE] = torrent->totalLeechersCount();
     const qreal ratio = torrent->realRatio();
     ret[KEY_TORRENT_RATIO] = (ratio > BitTorrent::TorrentHandle::MAX_RATIO) ? -1 : ratio;
+    ret[KEY_TORRENT_STATUS] = torrent->state().toString();
```

Comment:
If different code uses the same data in different ways we still don't need transfer many copies of it at once. Personally I can't imagine any strong reason for this behavior.